app/styles/main.css

```diff
@@ -197,3 +197,18 @@ body {
   text-overflow: ellipsis;
   margin-bottom: 20px;
 }
+
+.menu{ /*added another class so that the navigation links are in a box*/
```

Comment:
We don't need this class, we are going to use the bootstrap classes for the navigation bar.